experimental/packages/web-common/test/SessionLogRecordProcessor.test.ts

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as assert from 'assert';
+import { SessionLogRecordProcessor } from '../src/SessionLogRecordProcessor';
+import {
+  InMemoryLogRecordExporter,
+  LoggerProvider,
+  SimpleLogRecordProcessor,
+} from '@opentelemetry/sdk-logs';
+
+describe('SessionLogRecordProcessor', () => {
```

Comment:
nit: 
```suggestion
describe('SessionLogRecordProcessor', function () {
```

it's something that I've started doing everywhere when I write new tests in this repo, see the reason here: https://mochajs.org/#arrow-functions. It gives us more flexibility should we need to access the mocha context in the future. :slightly_smiling_face: